pkg/controller/service/service_controller_test.go

```diff
@@ -762,16 +760,16 @@ func TestServiceCache(t *testing.T) {
 		{
 			testName: "Set and Get",
 			setCacheFn: func() {
-				sc.set("addTest", &cachedService{state: defaultExternalService()})
```

Comment:
I'm wondering if it make sense for this to be shared across all the test tables? It seems like we're leaking state across test runs.

Reply:
Oh, I see, I have returned it. I just think it is not rigorous to judge empty only.